README.md

```diff
@@ -70,6 +92,36 @@ install.packages("FIELDimageR_0.2.0.tar.gz", repos = NULL, type="source") # Make
 
 <br />
 
+<div id="Instal_with_docker" />
+
+**With Docker**
+
+> When building the Docker image you will need the [Dockerfile](https://github.com/OpenDroneMap/FIELDimageR/blob/master/Dockerfile) in this repository available on the local machine.
+> Another requirement is that Docker is [installed](https://docs.docker.com/get-docker/) on the machine as well.
+
+> Open a terminal window and at the command prompt enter the following command to [build](https://docs.docker.com/engine/reference/commandline/build/) the Docker image:
+
+```bash
+docker build -t fieldimager -f ./Dockerfile ./
+```
+> The different command line parameters are as follows:
+>* `docker` is the Docker command itself
+>* `build` tells Docker that we want to build an image
+>* `-t` indicates that we will be specifying then tag (name) of the created image
+>* `fieldimager` is the tag (name) of the image and can be any acceptable name; this needs to immediately follow the `-t` parameter
+>* `-f` indicates that we will be providing the name of the Dockerfile containing the instructions for building the image
+>* `./Dockerfile` is the full path to the Dockerfile containing the instructions (in this case, it's in the current folder); this needs to immediately follow the `-f` parameter
+>* `./` specifies that Docker build should use the current folder as needed (required by Docker build)
+
+> Once the docker image is built, you use the [Docker run](https://docs.docker.com/engine/reference/run/) command to access the image using the suggested [rocker/rstudio](https://hub.docker.com/r/rocker/rstudio/) command:
+
+```bash
+docker run --rm -p 8787:8787 -e PASSWORD=yourpasswordhere fieldimager
```

Comment:
I propose setting (explicitly) the default username/password to rstudio/rstudio because it is somewhat of a convention (or perhaps the software's default?)

```suggestion
docker run --rm -p 8787:8787 -e USER=rstudio -e PASSWORD=rstudio fieldimager
```

Reply:
The suggested command in the dockerhub rocker/rstudio page uses `yourpasswordhere` (see https://hub.docker.com/r/rocker/rstudio/). Running the above command produced the following error:
```[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] add: executing... 
Nothing additional to add
[cont-init.d] add: exited 0.
[cont-init.d] userconf: executing... 


ERROR: You must set a unique PASSWORD (not 'rstudio') first! e.g. run with:
docker run -e PASSWORD=<YOUR_PASS> -p 8787:8787 rocker/rstudio


[cont-init.d] userconf: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] syncing disks.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

We can use a different password if that's desired

Reply:
Sorry - I'll defer to the rocker group for their guidance on default passwords!

Reply:
No worries. It is a good suggestion, it just didn't work out :(